package.json

```diff
@@ -13,9 +13,14 @@
     "LICENSE"
   ],
   "scripts": {
-    "build": "babel --out-dir lib src && cross-env BABEL_ENV=es babel --out-dir es src",
+    "prepublish": "npm run build",
+    "build": "rimraf lib && babel --out-dir lib src && cross-env BABEL_ENV=es babel --out-dir es src",
     "build-bower": "babel-node scripts/build-bower.js",
-    "test": "npm run build && mocha --compilers js:babel-core/register --require babel-polyfill \"test/**/*.spec.js\""
+    "prebuild-tsc": "rimraf built",
+    "build-tsc": "tsc --outDir lib",
+    "clean": "rimraf lib es built",
+    "tsc-errors-as-warnings": "echo \"TypeScript errors treated as warnings\"",
```

Comment:
The `tsc` command will return a non-zero status if it finds any errors, even though they are all really just warnings.   This unfortunately causes `npm` to get ugly.   

Before I had suppressed the errors, the `build-tsc` script read:  `tsc --outDir lib || tsc-errors-as-warnings` which stopped the ugliness.